client/src/components/Auth/SignIn.jsx

```diff
@@ -56,137 +54,151 @@ width: 70%;
 `;
 
 const StyledSvg = styled.svg`
-position: absolute;
+  position: absolute;
   bottom: 0;
   width: 100%;
   height: 50px;
-
 `;
 
 const StyledInput = styled.input`
-opacity: .5;
-// border-radius: 25px;
-border: none;
-border-bottom: .7px solid grey;
-color: grey;
-padding-left: 14px;
-margin-left: 10px;
-margin-top: 15px;
-margin-bottom: 15px;
-font-size: 18px;
-font-family: "Poppins", serif;
-text-align: left;
-height: 30px;
-background: none;
-// box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
-::placeholder: gold;
-width: 70%;
+  opacity: 0.5;
+  // border-radius: 25px;
+  border: none;
+  border-bottom: 0.7px solid grey;
+  color: grey;
+  padding-left: 14px;
+  margin-left: 10px;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  font-size: 18px;
+  font-family: "Poppins", serif;
```

Comment:
this font is from google fonts but I don't see it installed in the index.html file via cdn? Is this really being used or did someone just drop it in and forget to drop the cdn link in?

Reply:
Great question, I thought I had the cdn in there, but I suppose it's not and will be put in the next PR (though I'm not even sure what the team has agreed on for the font-family because I actually don't even remember "Poppins" as one of our options, ha.

Reply:
Added the CDN for 'Poppins' in case someone plans on using it